queries/leo/highlights.scm

```diff
@@ -0,0 +1,178 @@
+(variable) @variable
+
+[
+ "assert"
+ "assert_eq"
+ "assert_neq"
+ "block"
+ "console"
+ "let"
+ "mapping"
+ "program"
+ "record"
+ "self"
+ "struct"
+] @keyword
+
+[
+ "const"
```

Comment:
Hmm, const would be better suited as either `@type.qualifier` or `@storageclass` if it is similar to const in other languages like C, Rust, JS, etc